scripts/docker-compose/imports/openzaak-database/database/3-setup-indienen-aansprakelijkstelling-zaaktype.sql

```diff
@@ -902,10 +902,10 @@ VALUES
   'Besluit aansprakelijkstelling',
   'besluit-aansprakelijkstelling',
   'besluit',
+  'P2D',
```

Comment:
Maybe add a code comment here? Nobody will know what this does in a year's time I'm afraid.. ;-)

Reply:
Done. 

This should be documented on OpenZaak side as this format is enforced by them
On our side we use java.time.Period to parse these strings

Reply:
I know but this is our code base so we should also document for our future developers. :-)